packages/lodestar/src/network/util.ts

```diff
@@ -50,3 +55,105 @@ export async function initializePeerInfo(peerId: PeerId, multiaddrs: string[]):
   multiaddrs.forEach((ma) => peerInfo.multiaddrs.add(ma));
   return peerInfo;
 }
+
+export function encodeChunkifyResponse(config: IBeaconConfig, method: Method, {err, output}: ChunkResponse): Buffer {
+  if (err) {
+    return encodeResponseError(err);
+  }
+  let data = Buffer.alloc(0);
+  let type: AnyContainerType;
+  switch (method) {
+    case Method.Status:
+      type = config.types.Status;
+      break;
+    case Method.Goodbye:
+      type = config.types.Goodbye;
+      break;
+    case Method.BeaconBlocksByRange:
+    case Method.BeaconBlocksByRoot:
+      type = config.types.SignedBeaconBlock;
+      break;
+    default:
+      throw new Error(`Unhandled method ${method}`);
+  }
+  data = serialize(type, output);
+  return encodeSingleChunk(data);
+}
+
+export function decodeChunkifyResponse(config: IBeaconConfig, method: Method, chunks: Buffer): ResponseBody {
+  switch (method) {
+    case Method.Status:
+    case Method.Goodbye:
+      return decodeSingleChunk(config, method, chunks) as ResponseBody;
+    case Method.BeaconBlocksByRange:
+    case Method.BeaconBlocksByRoot:
+      // lodestar response multple single block chunks
+      // but other clients might response multiple n block chunks
+      // this handle both cases
```

Comment:
We should not be supporting multiple-block chunks.
According to the spec, "Each successful response_chunk MUST contain a single SignedBeaconBlock payload."